lib/include/jxl/decode.h

```diff
@@ -561,9 +569,9 @@ JXL_EXPORT void JxlDecoderCloseInput(JxlDecoder* dec);
  * @param dec decoder object
  * @param info struct to copy the information into, or NULL to only check
  * whether the information is available through the return value.
- * @return JXL_DEC_SUCCESS if the value is available,
- *    JXL_DEC_NEED_MORE_INPUT if not yet available, JXL_DEC_ERROR in case
- *    of other error conditions.
+ * @return @ref JXL_DEC_SUCCESS if the value is available,
+ *    @ref JXL_DEC_NEED_MORE_INPUT if not yet available, @ref JXL_DEC_ERROR in
+ * case of other error conditions.
```

Comment:
Indentation seems a bit messed up here

Reply:
Ugh, I blame clang-format. I'll fix this one.

Reply:
(also below)

I think I'd prefer sticking with indenting as it was before though.